src/client/sandbox/upload/hidden-info.ts

```diff
@@ -14,6 +14,8 @@ function createInput (form) {
     nativeMethods.inputValueSetter.call(hiddenInput, '[]');
     nativeMethods.appendChild.call(form, hiddenInput);
 
+    ShadowUI.markFormAsShadow(form);
```

Comment:
I guess we should leave `hiddenInput`  as the last child item of the `form`. It allows us to exclude its from loops (`for`,`for ...of`).